docs/BinSkimRules.md

```diff
@@ -870,3 +870,21 @@ This rule emits CSV data to the console for every compiler/language/version comb
 
 ---
 
+## Rule `BA6001.EnableSourceLink`
+
+### Description
+
+SourceLink information should be present in the PDB. This applies to binaries built with the C# and MSVC compilers. See https://aka.ms/sourcelink for more information.
+
+### Messages
+
+#### `Pass`: Pass
+
+The PDB for '{0}' contains SourceLink information.
+
+#### `Error`: Error
+
+The PDB for '{0}' does not contain SourceLink information.
```

Comment:
>The [](http://example.com/codeflow?start=0&length=3)

Our user-facing strings typically publish three data points, 1) the condition that prompted us to fire, 2) the risks associated with ignoring the result, 3) potential fixes.

We've got #1. I suppose the risks are increased engineering and security response costs when source code is not available for debugging. 

ERROR:
The PDB for '{0}' does not contain SourceLink information, increasing engineering and security response costs when source code isn't easily located for debugging; see [https://aka.ms/sourcelink](https://aka.ms/sourcelink) and [https://docs.microsoft.com/en-us/cpp/build/reference/sourcelink](https://docs.microsoft.com/en-us/cpp/build/reference/sourcelink) for information on how to enable.

Our pass messages need to broadcast core benefit:
PASS:

The PDB for '{0}' contains SourceLink information, maximizing engineering and security response cost efficiency when source code is required for debugging and other anlaysis.

Reply:
Done.